swagger/v1/get_moves_responses.json

```diff
@@ -15,7 +15,8 @@
         "type": "array",
         "items": {
           "anyOf": [
-            { "$ref": "location.json#/Location" },
+            { "$ref": "location.json#/Location1" },
```

Comment:
Hmm... I still think this is confusing the `Location` type with an example location. This endpoint will always return`Location` types; and that type is defined in the schemas part of the swagger file.